### PR TITLE
move kube-aws development to dedicated repository

### PR DESCRIPTION
**Pull request by colhom**
\cc @robszumski @mfburnett

### CODE REVIEW

**Comment by pieterlange** [comment]
@mumoshu released [0.9.0](https://github.com/coreos/kube-aws/releases/tag/v0.9.0) today, containing all the goodness from #544 #596 #608. This project really does fit better in it's own repo, thanks for all the work @colhom!

File: multi-node/aws/README.md [hunk 1]
```diff
@@ -1,210 +1,28 @@
...
 
+	|
+	|                *----------------*
+	|              *                    *
+	|            *                        *
+	|          *                            *
+	|        *                                *
+	|        |    ___  _____   ____    ____   |
+	|        |   |       |    |    |  |    |  |
+	|        |   |___    |    |    |  |____|  |
```
Comment (robszumski) [inline multi-node/aws/README.md]: Not sure I agree with this ASCII art. You will still be able to contribute to this repo...

Comment (pieterlange) [inline multi-node/aws/README.md]: This readme is located in the aws directory, so i think that's correct? New development is already taking place in https://github.com/coreos/kube-aws

Comment (robszumski) [inline multi-node/aws/README.md]: Oh I didn't catch the location. Disregard :)

**Pull request merged**